presto-docs/src/main/sphinx/release/release-329.rst

```diff
@@ -0,0 +1,65 @@
+=========================
+Release 329 (XXX 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix incorrect result for :func:`last_day_of_month` function for first day of month. (:issue:`2452`)
+* Fix incorrect results when handling ``DOUBLE`` or ``REAL`` types with ``NaN`` values. (:issue:`2582`)
+* Handle common disk failures during spill. When one disk fails, another
+  disk will be used automatically without failing the query. (:issue:`2444`)
+* Improve performance and reduce load on external systems when
+  querying ``information_schema``. (:issue:`2488`)
+* Limit broadcasted table size to ``100MB`` by default when using the ``AUTOMATIC``
+  join type selection strategy. (:issue:`2527`)
+* Hide non-standard columns ``comment`` and ``extra_info`` in the standard
+  ``information_schema.columns`` table. These columns can still be selected,
+  but will no longer appear when describing the table. (:issue:`2306`)
+* Enable :doc:`cost based </optimizer/cost-based-optimizations>` join reordering and join type selection
+  optimizations by default. The previous behavior can be restored by
+  setting ``optimizer.join-reordering-strategy`` configuration property to ``ELIMINATE_CROSS_JOINS``
+  and ``join-distribution-type`` to ``PARTITIONED``. (:issue:`2528`)
+
+Security Changes
+----------------
+
+* Add ``ldap.bind-dn`` and ``ldap.bind-password`` LDAP properties to allow LDAP authentication
+  access LDAP server using service account. (:issue:`1917`)
+
+Hive Connector Changes
+----------------------
+
+* Fix incorrect data returned when using S3 Select on uncompressed files. In our testing, S3 Select
+  was apparently returning incorrect results when reading uncompressed files, so S3 Select is disabled
+  for uncompressed files. (:issue:`2399`)
+* Fix incorrect data returned when using S3 Select on a table with ``skip.header.line.count`` or
+  ``skip.footer.line.count`` property. S3 Select API does not support skipping footers or more than one
+  line of a header.  In our testing, S3 Select was apparently sometimes returning incorrect results when
+  reading a compressed file with header skipping, so S3 Select is disabled when any of these table
+  properties is set to non-zero value. (:issue:`2399`)
+* Fix query failure for writes when one of the inserted ``REAL`` or ``DOUBLE`` values
+  is infinite or ``NaN`` and column statistics collection is enabled. (:issue:`2471`)
```

Comment:
Originally i worded this as "and column statistics collection is not disabled", because it's enabled by default.

Maybe we should just omit that part?

Reply:
The phrase "is not disabled" sounds like a double negative, but I see your point. Omit sounds good.